### PR TITLE
caf 0.18.2

### PR DESCRIPTION
**Pull request by Neverlord**
Update CAF to version 0.18.2

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
```
==> Downloading https://github.com/actor-framework/actor-framework/archive/0.18.2.tar.gz
Already downloaded: /Users/brew/Library/Caches/Homebrew/downloads/3820189d9a21c0ad766a5335fa33f361f36561abde630fca16acb273ed0cac79--actor-framework-0.18.2.tar.gz
Error: SHA256 mismatch
Expected: cb55a089ac468d5d526a10259ffcafda2d5a9ad0513bfbe64088d39c2c955c64
  Actual: 03837dbef73a13c3e4cac76c009d018f48d1d97b182277230f204984c9542ccc
```
They might've re-tagged the release.

File: Formula/caf.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Caf < Formula
   # Renamed from libccpa
   desc "Implementation of the Actor Model for C++"
   homepage "https://www.actor-framework.org/"
-  url "https://github.com/actor-framework/actor-framework/archive/0.18.1.tar.gz"
-  sha256 "9d9ba0c4bf41ce7a1d7cae9a0efa83816a610b5e84077a02d02cffb8513a5b6d"
+  url "https://github.com/actor-framework/actor-framework/archive/0.18.2.tar.gz"
+  sha256 "cb55a089ac468d5d526a10259ffcafda2d5a9ad0513bfbe64088d39c2c955c64"
```
Comment (carlocab) [inline Formula/caf.rb:6]: Won't you still get a checksum mismatch here?
```
❯ curl -sSL https://github.com/actor-framework/actor-framework/archive/0.18.2.tar.gz | sha256sum
03837dbef73a13c3e4cac76c009d018f48d1d97b182277230f204984c9542ccc  -
```

Comment (Neverlord) [reply Formula/caf.rb]: Yes, you're right. Sorry for the noise.

**Comment by carlocab** [comment]
Looks like you accidentally included a bunch of extra (now deleted) formulae in d0adcbfbe33ef4e91bad752faa448f73379024cc. I cleaned that up for you.

**Review by carlocab — APPROVED** [review]

**Comment by Neverlord** [comment]
> Looks like you accidentally included a bunch of extra (now deleted) formulae in [d0adcbf](https://github.com/Homebrew/homebrew-core/commit/d0adcbfbe33ef4e91bad752faa448f73379024cc). I cleaned that up for you.

Weird, I've used `bump-formula-pr` for this. Anyway, thanks a lot! 🙂

**Comment by carlocab** [comment]
This is the commit created by `bump-formula-pr`: a1936f7886753e4cdce623153216ab9b4c1e1822

You must've picked up the extra ones while rebasing to fix the checksum.

**Pull request closed without merging**